### PR TITLE
Minor improvement on code logic

### PR DESCRIPTION
**Pull request by yifeili98**

### CODE REVIEW

**Comment by yifeili98** [comment]
Next commit will combine some functions with similar functionalities.

File: PortalFetch/crawler_download.py [hunk 1]
```diff
@@ -3,277 +3,246 @@
...
-    time.sleep(2)  # Waiting for completion
-
-
-def advanceSearch(driver):
-    """
-    Locate 'Advanced Search' button and click in the 'Look Up Classes' page.
+from selenium.common.exceptions import ElementNotVisibleException
+from crawler_data_process import DataProcess
+
+logging.basicConfig(filename = '../log/' +
```
Comment (kis87988) [inline PortalFetch/crawler_download.py:20]: maybe we should create a logger package in general. Then we just need to do pip install.

**Review by kis87988 — COMMENTED** [review]
let's keep the original logic comment too.

**Comment by kis87988** [comment]
Let's put the logic comment back to `PortalFetch/crawler_download.py`

File: PortalFetch/crawler_download.py [hunk 2]
```diff
@@ -3,277 +3,252 @@
...
+    raise ElementNotVisibleException("Could not load the full page!")
 
 
 def main():
-    """
-    Download course information from De Anza myportal.
+    """Download course information from De Anza myportal.
 
     Login in De Anza myportal using username and password.
     click Apps-Lookup Classes-Select by term -submit-Advanced Search-in Subject, select all-Section search-Download all the course infromation-Save in an excel
```
Comment (kis87988) [inline PortalFetch/crawler_download.py:210]: is this line not too long?

File: PortalFetch/crawler_download.py [hunk 3]
```diff
@@ -3,277 +3,252 @@
...
+            list_group_item = driver.find_element_by_class_name("list-group-item")
+        except:
+            pass
+        web_driver_counter -= 1
+    if not list_group_item:
+        logger.error("Could not find list-group item!")
+        raise NoSuchElementException("Could not find list-group item!")
+
+    try:
+    	# Course search page from homepage after login
```
Comment (kis87988) [inline PortalFetch/crawler_download.py:229]: indent

**Review by kis87988 — APPROVED** [review]

**Review by biankaiwen111 — APPROVED** [review]
LGTM!

**Pull request merged**